SmartPlaces.Facilities/lib/OntologyMapper.Mapped/test/WillowMappingValidationTests.cs

```diff
@@ -94,5 +95,49 @@ public void ValidateMappedDtmisAreValidFormat(string resourcePath)
 
             Assert.Empty(exceptions);
         }
+        
+        // TODO: This test needs a new version of the Mapped DTDL to get it to work.
```

Comment:
Should we hold off on placing placeholder code until its ready and we know it operates like with expect it to?

Reply:
I expect to have it within the day... so I'll leave it for now.